cpython/Makefile

```diff
@@ -15,6 +15,12 @@ URL=https://www.python.org/ftp/python/$(PYVERSION)/Python-$(PYVERSION).tgz
 LIB=libpython$(PYMINOR).a
 
 
+ZLIBVERSION = 1.2.11
+ZLIBTARBALL=$(ROOT)/downloads/zlib-$(ZLIBVERSION).tar.gz
+ZLIBBUILD=$(ROOT)/build/zlib-$(ZLIBVERSION)
+ZLIBURL=https://zlib.net/zlib-1.2.11.tar.gz
```

Comment:
Python used to ship its own copy of `zlib`.  It no longer does, so now we'll download and use our own.